Makefile

```diff
@@ -220,6 +219,12 @@ build-operator-image: ## Build the operator image.
 	--build-arg VCS_REF=$(VCS_REF) --build-arg VCS_URL=$(VCS_URL) \
 	--build-arg GOARCH=$(LOCAL_ARCH) -f Dockerfile .
 
+build-operator-dev-image: ## Build the operator dev image.
+	@echo "Building the $(OPERATOR_IMAGE_NAME) docker image..."
+	@docker build -t $(OPERATOR_IMAGE_NAME):$(VERSION) \
+	--build-arg VCS_REF=$(VCS_REF) --build-arg VCS_URL=$(VCS_URL) \
+	--build-arg GOARCH=$(LOCAL_ARCH) -f Dockerfile .
+
 build-test-operator-image: ## Build the operator test image.
 	@echo "Building the $(OPERATOR_IMAGE_NAME) docker image for testing..."
 	@docker build -t $(QUAY_REGISTRY)/$(OPERATOR_IMAGE_NAME):$(OPERATOR_TEST_TAG) \
```

Comment:
Also `build-test-operator-image` is used for `e2e-test-kind`, and the image is using `OPERATOR_TEST_TAG` which is `dev-test` in line 83. Could we change the tag to `e2e-test` to distinguish it from `make build-operator-dev-image`?

Reply:
I believe we should separate them. `build-operator-dev-image` builds the image for developers. `build-test-operator-image` builds the image for CICD.

It keeps flexibility for future change.

Reply:
Sure, we need to keep them separately.